### ISSUE TITLE
Request: animate without targets

### ISSUE THREAD

**Issue by DarKDinDoN** [otigon/automated-jb2a-animations#18]
Hello,

Might be a wild suggestion 😄 
For those of us who play without automation (better rolls, default 5e workflow, etc.), we don't use the targeting system for each and every attack/spell. My players often just tell me "I attack the foe in front of me" without using the targeting tool.

In this case, the animation wouldn't play. Would it be possible to play the animation anyway?
* The dagger is thrown in front of the token and stops mid-air (without hitting an actual target)
* Same for spell
* Melee attacks animations are played in front of the token as well

Even tho, that might be weird to have an animation playing in the wrong direction of not hitting something, that would be awesome enough to see the animation playing.

Furthermore, a targeting system that request the user to click on one or multiple tokens could be an option as well.
* The user clicks on an ability, attack, spell
* The user clicks on a target (without having to select the targeting tool)
* Then and only after then the dices are thrown and the result is shown in the chat

**Comment by otigon** [comment]
Something along these lines has been requested often. I'm still chewing on it and trying to decide how, if at all possible, to implement it. As far as Foundry is concerned, "targeted" and "selected" tokens are completely different, and the majority of these animations require both to tell FX Master where to play the animation on the Canvas.

Defaulting to "some" animation being played is more doable, especially with the Melee attacks. This is something I should be able to implement soon. However, ranged attacks are a different beast. There are generally 3 different sizes of animations depending on distance to target because I have to stretch/shrink them to get them to play properly.

Long story short, defaulting to playing an animation in a set direction is doable and I do plan to put that in at some point. Requesting a target afterwards is probably not possible because, imo, there are too many failure points along the way for that to get out of control.

**Comment by otigon** [comment]
For attack spells and weapons, this will unfortunately always need a target
